services/google_scraping_service/job_enqueuer.go

```diff
@@ -0,0 +1,38 @@
+package google_scraping_service
+
+import (
+	"errors"
+	"log"
+
+	"github.com/gocraft/work"
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/models"
+)
+
+const (
+	invalidKeyword = "invalid keyword"
+)
+
+var EnqueueScrapingJob = func(savedKeyword models.Keyword) error {
```

Comment:
Related to the new name I am recommending, this method name could be changed to:

```suggestion
var EnqueueSearchJob = func(savedKeyword models.Keyword) error {
```

Reply:
Fixed in 5ec85f0f67f6e6f0b2e9d7839120284ac2b90be0